langchain/agents/chat/prompt.py

```diff
@@ -0,0 +1,18 @@
+# flake8: noqa
+PREFIX = """Answer the following questions as best you can. You have access to the following tools:"""
+FORMAT_INSTRUCTIONS = """The way you use the tools is by specifying a json blob.
+Specifically, this json should have a `action` key (with the name of the tool to use) and a `action_input` key (with the input to the tool going here).
```

Comment:
```suggestion
Specifically, this json should have a `action` key (with the name of the tool to use) and a `action_input` key (with the input to the tool going here).
```

If we didn't tell them that it's a string literal, sometimes an object would be entered and processing could not continue.